tracer/src/Datadog.Tracer.Native/iast/method_info.cpp

```diff
@@ -415,7 +415,10 @@ namespace iast
             verificationFail = analysis.GetError();
             if (!correct || dump)
             {
-                analysis.Dump(isRejit ? "ReJit " : "  Jit ");
+                std::string message = !correct ? " [ IL Verification FAILED. Discarded ] "
+                                               : " [ Written Correctly ] " + 
```

Comment:
nit: can we reverse this condition? 😄 

```suggestion
                std::string message = correct ? " [ Written Correctly ] "
                                               : " [ IL Verification FAILED. Discarded ] " + 
```